apigateway.tf

```diff
@@ -62,9 +62,10 @@ resource "aws_api_gateway_domain_name" "domain" {
 }
 
 resource "aws_api_gateway_deployment" "stage" {
-  count       = local.use_api_gateway
-  rest_api_id = aws_api_gateway_rest_api.api-gateway[0].id
-  stage_name  = var.environment
+  count           = local.use_api_gateway
+  rest_api_id     = aws_api_gateway_rest_api.api-gateway[0].id
+  stage_name      = var.environment
+  security_policy = var.security_policy
```

Comment:
are you sure this resource have this parameter?
I dont see it in https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/api_gateway_deployment